email_account_validity/_store.py

```diff
@@ -261,12 +290,14 @@ def set_renewal_token_for_user_txn(txn: LoggingTransaction):
         )
 
     async def get_user_from_renewal_token(
-        self, renewal_token: str
+        self, renewal_token: str, user_id: Optional[str] = None,
     ) -> Tuple[str, int, Optional[int]]:
```

Comment:
this is somewhat misnamed now that sometimes you have to give it a user_id. I think maybe it would be clearer to have two separate methods?

Reply:
I think it's clearer to have one function (with a less confusing name) rather than two functions that do essentially the same thing.